src/tmb_includes.h

```diff
@@ -1,5 +1,55 @@
 #ifndef TMB_INCLUDES_H
 #define TMB_INCLUDES_H
+#ifdef INCLUDE_RCPP
+/* Rcpp defines `R_NO_REMAP` which causes errors when using TMB. Thus, we
```

Comment:
hm all of this is not really nice. is there a way we could instead of doing this here contribute to `TMB` project to have this managed there?

Reply:
need further investigation here; the Rcpp and TMB stuffs are quite complicated

Reply:
yeah you can file a separate issue for that then we can follow up later.

Reply:
@clarkliming please file a follow-up issue so that we don't forget this, thanks